### ISSUE TITLE
Deprecations in Mendix 6.9

### ISSUE THREAD

**Issue by David-Sanders** [Mcpoowl/SFTPModule#7]
The SFTP module is giving 5 deprecation warnings in v6.9 as follows:

'Traversing association 'SFTP.Document_Connection' from persistable entity 'SFTP.Connection' to non-persistent entity 'SFTP.Document' is deprecated.  In Mendix 7 this will no longer be allowed'

So it looks like the module will no longer work when Mendix 7 is released

**Comment by Mcpoowl** [comment]
That's true David. During one of the Mendix Meetups we discussed the SFTP module, and noticed there are a couple of versions lying around from different Partners. Some guys from Capgemini told us they were working on a remaster of the SFTP module, with some enhancements aswell. We talked about that version being the one and only SFTP module we should need, ever.

However, since that meetup I've never received any communication regarding the outcome / progress of the module.

As for this version, I'm not sure when I have the time, but when I do I'll see if I can change some things around.

**Comment by RCatersels** [comment]
Are we able to contact the Capgemini guys? They have my businesscard but they failed to contact me. My goal would still be to have one SFTP module.

**Comment by Mcpoowl** [comment]
I have no way to contact them, and they don't have my information either. I agree one module would be the best

**Comment by RCatersels** [comment]
I have sended a message to Christiaan Tick who seems to be the owner on de Capgemini side.

**Comment by David-Sanders** [comment]
I believe this is the (S)FTP module from Pink Elephant.  The Cap Gemini one does not seem to be on Github.  One simple solution seems to be to make the Connections entity also non-persistent, and remove the link to the account object, then change the microflows that retrieve/create it to always create a new record in the NP entity

**Comment by RCatersels** [comment]
I have been in contact with Adnan Ramlawi from Cap Gemini. He will be responsible to publish their version on Github. Since Paul and myself do not care under which flag the FTP module is published we all will start contributing on one unified module. Cap Gemini (Adnan) will be responsible to transfer the github versions to the Mendix appstore. I will see Adnan today (2-11-2016) at the  Mendix meetup at Mendix HQ.
